src/Common/src/System/Security/Cryptography/DerSequenceReader.cs

```diff
@@ -194,6 +194,30 @@ internal string ReadOidAsString()
             return builder.ToString();
         }
 
+        internal byte[] ReadBitString()
+        {
+            // Bitstrings are primitives with tag and length, followed by a number
+            // of unused bits in the last byte, followed by the 
+            EatTag(DerTag.BitString);
+            int length = EatLength();
+
+            if (length == 0)
+                return Array.Empty<byte>();
+
+            // We skip over the number of unused bits as it is not needed, and allocate one less 
+            // byte as this is not included in the bitstring. According to DER, all trailing padding must be
+            // composed of zeros, so there's no need to clear these bits. In case this encoder is expanded to BER
+            // or other encodings it would be necessary to read this number and clear the corresponding bits.
```

Comment:
Modulo lining the data up into the registers in the first place, I think it can be made primitive-encoding-BER-capable in 4 assembly instructions (I feel like I'm coming up with 8ish overall?).  So it doesn't seem overly onerous.